models/UserManager.php

```diff
@@ -0,0 +1,336 @@
+                }
+
+                if(!empty($errors)){
+                    ?>
+                        <div class="card red">
+                            <div class="card-content white-text">
+                                <?php
+                                foreach($errors as $error){
+                                    echo $error;
+                                }
+                                ?>
+                            </div>
+                        </div>
+                    <?php
+                }else{
+                    Usermanager::add_modo($name,$email,$role,$token);
+                    header("Location:/gestion-des-admins-et-modos");
+                }
+            }
+    }
+    
+    function new_verification()
+    {
+        
+                        if(filter_has_var(INPUT_POST, 'submit')){
+                    $email = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'email')));
+                    $token = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'token')));
+
+                    $errors = [];
+
+                    if(empty($email) || empty($token)){
+                        $errors['empty'] = "Tous les champs n'ont pas été remplis";
+                    }else if(UserManager::is_modo($email,$token) == 0){
+                        $errors['exist'] = "Ce modérateur n'existe pas";
+                    }                
+
+                    if(!empty($errors)){
+                        ?>
+                        <div class="card red">
+                            <div class="card-content white-text">
+                                <?php
+                                foreach($errors as $error){
+                                    echo $error;
+                                }
+                                ?>
+                            </div>
+                        </div>
+                    <?php
+                    }else{
+                        $_SESSION['admin'] = $email;
+                        header("Location:/modification-du-mot-de-passe");
+                    }
+                }
+    }
+    
+    function password_verification()
+    {
+   if(filter_has_var(INPUT_POST, 'submit')){
+                    $password = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'password')));
+                    $password_again = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'password_again')));
+
+                    $errors = [];
+                    if(empty($password) || empty($password_again)){
+                        $errors['empty'] = "Tous les champs n'ont pas été remplis";
+                    }
+
+                    if($password != $password_again){
+                        $errors['different'] = "Les mots de passe sont différents";
+                    }
+
+                    if (preg_match('#^(?=.*[a-z])(?=.*[A-Z]).{8,}$#', $password)) {
+                    }
+                    else{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method password_verification uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)